src/ape/utils.py

```diff
@@ -216,6 +296,16 @@ def extract_nested_value(root: Mapping, *args: str) -> Optional[Dict]:
 
 
 def stream_response(download_url: str, progress_bar_description: str = "Downloading") -> bytes:
+    """
+    To prvoide a visual represenation of the downloads.
```

Comment:
A visual representation is part of it, but this method really is just in charge of streaming an HTTP response
```suggestion
    Download HTTP content by streaming and returning the bytes.
```

Reply:
you can also mention that progress bar will display in the CLI